scripts/common/proxy.sh

```diff
@@ -1,25 +1,78 @@
 
+function proxy_bootstrap() {
```

Comment:
We need to know the proxy address to download the binaries but we don't have the binaries to parse the yaml spec so try to find it with grep.

Reply:
This is also an issue with airgap, it currently will only work as a bash flag and not as yaml spec because the airgap stuff happens before the yaml is parsed. Open to suggestions to fixing this.

Reply:
We could add an empty file to every airgap bundle named `airgap` and if that file exists it's an airgap install. Or, since the bin/ utils will already be there on airgap installs, we could try to use them to parse the yaml for the airgap flag and if all that succeeds it's airgap.